src/tests/DateRangePicker.spec.js

```diff
@@ -340,43 +335,44 @@ describe('The DateRangePicker component', function () {
     describe('each component is provided with a number of event handlers', function () {
 
       it('onMonthChange calls #changeMonth', function () {
-        var spy = spyOn(DateRangePicker.prototype.__reactAutoBindMap, 'changeMonth');
-        this.useDocumentRenderer();
+        this.useDocumentRenderer({
+          initialMonth: 6,
+        });
         var calendarMonthComponent = TestUtils.scryRenderedComponentsWithType(this.renderedComponent, CalendarMonth)[0];
         calendarMonthComponent.props.onMonthChange(5);
-        expect(spy).toHaveBeenCalled();
+        expect(this.renderedComponent.state.month).toBe(5);
       });
 
       it('onYearChange calls #changeYear', function () {
-        var spy = spyOn(DateRangePicker.prototype.__reactAutoBindMap, 'changeYear');
-        this.useDocumentRenderer();
+        this.useDocumentRenderer({
+          initialYear: 2000,
```

Comment:
I do not think so. This would test that a prop is correctly applied initially to the component state. Looks this might have been overlooked in the initial implementation :)